tests/integration.rs

```diff
@@ -7284,16 +7309,30 @@ mod live_migration {
                 "guest_numa_id=0,cpus=[0-2,9],distances=[1@15,2@20],memory_zones=mem0",
                 "guest_numa_id=1,cpus=[3-4,6-8],distances=[0@20,2@25],memory_zones=mem1",
                 "guest_numa_id=2,cpus=[5,10-11],distances=[0@25,1@30],memory_zones=mem2",
-            ]
-        } else if local {
-            &["--memory", "size=4G,shared=on"]
-        } else {
-            &["--memory", "size=4G"]
+            ],
+            (true, false) => &["--memory", "size=4G,shared=on"],
+            (true, true) => &[
+                "--memory",
+                "size=0,hotplug_method=virtio-mem,shared=on",
+                "--memory-zone",
+                "id=mem0,size=1G,hotplug_size=32G,shared=on",
+                "id=mem1,size=1G,hotplug_size=32G,shared=on",
+                "id=mem2,size=2G,hotplug_size=32G,shared=on",
+                "--numa",
+                "guest_numa_id=0,cpus=[0-2,9],distances=[1@15,2@20],memory_zones=mem0",
+                "guest_numa_id=1,cpus=[3-4,6-8],distances=[0@20,2@25],memory_zones=mem1",
+                "guest_numa_id=2,cpus=[5,10-11],distances=[0@25,1@30],memory_zones=mem2",
+            ],
```

Comment:
Make sure this part (where you extend the existing test) is part of a separate commit.